src/core/actions/qgsaction.cpp

```diff
@@ -34,26 +34,44 @@
 #include "qgsvectorlayer.h"
 #include "qgslogger.h"
 #include "qgsexpressioncontextutils.h"
-#include "qgswebview.h"
 #include "qgsnetworkaccessmanager.h"
 #include "qgsmessagelog.h"
 #include "qgsvariantutils.h"
 
 bool QgsAction::runable() const
 {
-  return mType == Generic ||
-         mType == GenericPython ||
-         mType == OpenUrl ||
-         mType == SubmitUrlEncoded ||
-         mType == SubmitUrlMultipart ||
+#ifndef __clang_analyzer__
```

Comment:
Could we use NOLINT instead with the specific check we want to avoid ? https://clang.llvm.org/extra/clang-tidy/#suppressing-undesired-diagnostics

I don't see why you get the bugprone-branch-clone warning check, Q_OS_WIN and Q_OS_MAC should not be defined and everything should be right.